config.yaml

```diff
@@ -41,4 +37,12 @@ options:
   userid-claim:
     type: string
     default: 'email'
-    description: OpenID Connect claim whose value will be used as the userid.
\ No newline at end of file
+    description: OpenID Connect claim whose value will be used as the userid.
+  public-url:
```

Comment:
Isn't keeping the `public-url` config option misleading as long as we 've removed the corresponding code from the charm? I mean, using it won't do anything right now.

Reply:
Yeah, thinking about it more, I don't see a reason to keep it other than it being informative that the config option is deprecated.